src/Neo.VM/ExecutionEngine.cs

```diff
@@ -59,7 +59,7 @@ public class ExecutionEngine : IDisposable
         /// <summary>
         /// The VM object representing the uncaught exception.
         /// </summary>
-        public StackItem? UncaughtException { get; internal set; }
+        public StackItem? UncaughtVMCatchableException { get; internal set; }
```

Comment:
I think that we don't need to rename this one

Reply:
not a problem, if you think this pr is ok, will revert it.

Reply:
Yes, you want to throw only VMXXExceptions, ins't it? maybe we should have and interface for valid exceptions

Reply:
something like `IVMException`